docs/notebooks/meta_learners_synthetic_data.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Maybe add https://matheusfacure.github.io/python-causality-handbook/20-Plug-and-Play-Estimators.html as a reference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>